app/models.py

```diff
@@ -0,0 +1,25 @@
+from pydantic import BaseModel
+from enum import Enum
+
+class Facetname(str, Enum):
```

Comment:
Facetname should be in singular for how we will use them.

https://world.openfoodfacts.org/countries/ for all countries

But **country** for a single country : https://world.openfoodfacts.org/country/india
(and your urls are for single values)